### PR TITLE
Add release notes item about the JNA version bump

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this is the missing release notes item for e7022d6. Please let me know if that item should go to the "Fixes" section instead.

With kind regards,
Andreas.

### CODE REVIEW

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -97,6 +97,9 @@ Changes
   where there are filters restricting the result set or if only a sub-set of
   the columns is selected.
 
+- Bump JNA library to version 5.6.0. This will make CrateDB start flawlessly
```
Comment (seut) [inline docs/appendices/release-notes/unreleased.rst:100]: As this will be backported and is treat as a fix, it must go into the `Fixes` section

Comment (amotl) [reply docs/appendices/release-notes/unreleased.rst]: Thanks. I've amended the patch.

**Review by seut — APPROVED** [review]

**Comment by mkleen** [comment]
The changelog in the release file is usually written in the past not in the present.

**Comment by amotl** [comment]
Thanks, I've addressed all your suggestions.

**Pull request merged**